docs/blog/posts/2024-04-25-yen_gpu.md

```diff
@@ -11,69 +11,60 @@ subtitle: Running Jobs on Yen's GPU node
 
+- `yen-gpu1` has 64 threads, 256 G of RAM and 4 A30 NVIDIA GPU's
+- `yen-gpu2` and `yen-gpu3` each have 64 threads, 256 G of RAM and 4 A40 NVIDIA GPU's
 
-<!-- more -->
-## Login to Yens
+The A30 NVIDIA GPU's have 24 G of GPU RAM while the A40 NVIDIA GPU's have 48 G of GPU RAM.
 
-Enter your SUNet ID and Duo authenticate to login.
+### GPU partition
 
-```bash
-$ ssh <$USER>@yen.stanford.edu
-```
-
-### Slurm GPU partition
-
-The `yen-slurm` cluster now has a new `gpu` partition to run jobs on the GPU nodes. See its timelimit with:
+To work with these GPU nodes on Yen Slurm, you can [submit jobs](/_user_guide/slurm/#how-do-i-use-the-scheduler){:target="_blank"} to the `yen-slurm` cluster targeting the `gpu` partition. You can use the command `sinfo -p gpu` to get more information about this partition:
 
-```bash
-$ sinfo -p gpu
+```{.yaml .no-copy title="Terminal Output"}
 PARTITION AVAIL  TIMELIMIT  NODES  STATE NODELIST
-gpu          up 1-00:00:00      2   idle yen-gpu[1-3]
+gpu          up 1-00:00:00      1    mix yen-gpu3
+gpu          up 1-00:00:00      2   idle yen-gpu[1-2]
 ```
 
 !!! warning
     There is a limit of 1 day runtime and 2 GPU's per user.
 
-## Python Example
-This guide will detail how to run a short Python example using <a href="https://pytorch.org/" target="_blank">PyTorch</a> or
- <a href="https://keras.io/about/" target="_blank">Keras</a> for deep learning training. 
-CUDA, PyTorch and Tensorflow/Keras are installed already so you do not have to install them yourself.
+## Usage Example with Python
+This guide will detail how to run a short Python example using both [PyTorch](https://pytorch.org/){:target="_blank"} and [Keras](https://keras.io/about/){:target="_blank"} for deep learning training. CUDA, PyTorch and Tensorflow/Keras are installed already so you do not have to install them yourself.
 
-### PyTorch Example
+### Using PyTorch
 
-To use `PyTorch`, you simple load the `pytorch` module and run the training example.
+To use PyTorch, you can simply load the `pytorch` module pre-installed on the system.
 
-```bash
-$ ml pytorch
+```bash title="Terminal Command"
+ml pytorch
 ```
 
-When you load this module, you will be in a python `venv` running Python 3.10 with pytorch and other python packages installed in that `venv` already.
-
-You can check with:
+Once this module is loaded, you will be in a Python 3.10 `venv` running with `pytorch` and other relevant packages installed in that virtual environment already.
 
-```bash
-$ pip list
+You can check this by running `pip list`, which will list all of the packages installed in the environment. The below output shows some relevant packages:
 
-$ python --version
-Python 3.10.12
+```{.yaml .no-copy title="Terminal Output"}
+...
+nvidia-cuda-runtime-cu12  12.1.105
+...
+pytorch-lightning         2.1.3
+...
 ```
 
-If you need additional packages installed, you can `pip install` them to your `~/.local` since this global `venv` 
```

Comment:
is this what we recommend?

Reply:
hmmmm, i dont know what is the best practice here besides creating your own copy of the environment, perhaps i will remove it for now and we can make another blog post about it in the future when we know whats best